pyanaconda/modules/payloads/payload/dnf/dnf.py

```diff
@@ -54,11 +56,21 @@ def supported_source_types(self):
 
     def process_kickstart(self, data):
         """Process the kickstart data."""
-        pass
+        source_type = SourceFactory.get_rpm_type_for_kickstart(data)
+
+        if source_type is None:
+            return None
```

Comment:
The method `process_kickstart` doesn't return anything.

Reply:
Good point. Thanks.